app/models/user.rb

```diff
@@ -19,7 +19,15 @@ class User < ApplicationRecord
   scope :halfyear_actives, ->(time = Time.now) { logged_in_since(time - 6.month) }
   scope :active, -> { joins(:person).where(people: {closed_account: false}) }
 
-  devise :database_authenticatable, :registerable,
+  attribute :otp_secret
+
+  devise :two_factor_authenticatable,
+         :two_factor_backupable,
+         otp_secret_encryption_key: Diaspora::Application.config.twofa_encryption_key,
```

Comment:
Oh, this needs to be `AppConfig.twofa_encryption_key` then.

Reply:
Ah, nice!